--- setup.py.bak	2026-03-16 00:11:39.050608138 +0000
+++ setup.py	2026-03-16 00:14:14.397696340 +0000
@@ -6,11 +6,17 @@
 # it, since setuptools is now required at build-time
 import setuptools
 
-
 sources = ['src/c/_cffi_backend.c']
 libraries = ['ffi']
-include_dirs = ['/usr/include/ffi',
-                '/usr/include/libffi']    # may be changed by pkg-config
+
+_prefix = os.environ.get('PKG_CONFIG_SYSROOT_DIR', '/')
+_include_dirs = ['usr/include/ffi', 'usr/include/libffi']
+
+# may be changed by pkg-config
+include_dirs = []
+for p in _include_dirs:
+    include_dirs.append(os.path.join(_prefix, p))
+
 define_macros = [('FFI_BUILDING', '1')]   # for linking with libffi static library
 library_dirs = []
 extra_compile_args = []
